js/main.js

```diff
@@ -1,16 +1,184 @@
-const getRandom = function(min, max, floatNumber){
-  if(min < 0 || max < 0 || floatNumber < 0){
+function getRandomPositiveNumber() {
+  let number = Math.floor(Math.random() * 100);
+  while (number < 0){
+    number = Math.floor(Math.random());
+  }
+  return number;
+}
+
+function getRandomPositiveInteger (a, b) {
+  if (a < 0 || b < 0) {
+    return NaN;
+  }
+
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+
+  const result = Math.random() * (upper - lower + 1) + lower;
+
+  return Math.floor(result);
+}
+
+
+function getRandomPositiveFloat (a, b, digits) {
+  if (a < 0 || b < 0 || digits < 0) {
     return NaN;
-  } else if (min > max){
-    const c = min;
-    min = max;
-    max = c;
-  } else if (min === max){
-    return min;
   }
+  const lower = Math.min(a, b);
+  const upper = Math.max(a, b);
+
+  const result = Math.random() * (upper - lower) + lower;
+
+  return +result.toFixed(digits);
+}
+
+function getRamdomArrayElement (elements) {
+  return elements[getRandomPositiveInteger(0, elements.length - 1)];
+}
+
+function shuffle(array) {
+  let currentIndex = array.length, randomIndex;
+
+  // While there remain elements to shuffle.
+  while (currentIndex !== 0) {
+
+    // Pick a remaining element.
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex--;
+
+    // And swap it with the current element.
+    [array[currentIndex], array[randomIndex]] = [
+      array[randomIndex], array[currentIndex]];
+  }
+  return array;
+}
 
-  const randomNumbder = (Math.random() * (max - min) + min).toFixed(floatNumber);
-  return parseFloat(randomNumbder);
+const getRandomShuffledArray = function(array) {
+  const index = getRandomPositiveInteger(0, array.length - 1);
+  array = shuffle(array);
+  const shuffled = array.slice(0, index);
+  return shuffled;
 };
 
-getRandom(2, 120, 2);
+//  location
+// const lat = getRandomPositiveFloat(35.65, 35.7, 5);
+// const lng = getRandomPositiveFloat(139.7, 139.8, 5);
+
+// const createLocation = () => {
+// return lat + ", " + lng;
+// };
+
+function getRandomPositiveIntegerWithZero (a, b) {
+  if (a < 0 || b < 0) {
+    return NaN;
+  }
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+  const result = Math.floor(Math.random() * (upper - lower + 1) + lower);
+  if (result < 10) {
+    return `0${result}`;
+  }
+  return result;
+}
+
+// Author
+function createAuthor() {
+  return `img/avatars/user${getRandomPositiveIntegerWithZero(1,10)}.png`;
+}
+
+// Offer
+const TITLE = [
```

Comment:
```suggestion
const TITLES = [
```
Тут и далее, массивы пишутся в верхнем регистре